src/www/ui_components/app-root.js

```diff
@@ -47,6 +47,7 @@ import './outline-icons.js';
 import './privacy-view.js';
 import './server-connection-viz.js';
 import './server_list.ts';
+import './server_card.ts';
```

Comment:
Why are these files using underscores?  I think everything else is using hyphens.

Reply:
Our style guide wants underscores